GiniVision/Classes/CameraViewController.swift

```diff
@@ -290,6 +311,36 @@ public typealias CameraScreenFailureBlock = (_ error: GiniVisionError) -> ()
         cameraOverlay.alpha = 0
     }
     
+    fileprivate func showFileImportTip() {
```

Comment:
I'd extract this + the user defaults stuff above to a new file.

Reply:
I extracted the user defaults stuff to the ToolTipView. I had to move the creation of the tip to other method since it could be the case that the onboarding appears and the tooltip has to be hidden, so it will be shown when the onboarding dismisses.

And the creation of the tip cannot be moved to other view since the blurred view depends on it, also I wouldn't move it to other file because all the views on the CameraViewController are created within the class.